self-hosted-login/okta-aspnetcore-mvc-example/Startup.cs

```diff
@@ -31,19 +31,19 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             services.AddAuthentication(options =>
-            {
-                options.DefaultAuthenticateScheme = CookieAuthenticationDefaults.AuthenticationScheme;
-                options.DefaultSignInScheme = CookieAuthenticationDefaults.AuthenticationScheme;
-                options.DefaultChallengeScheme = OktaDefaults.MvcAuthenticationScheme;
-            })
-            .AddCookie()
-            .AddOktaMvc(new OktaMvcOptions
-            {
-                // Replace these values with your Okta configuration
-                OktaDomain = Configuration.GetValue<string>("Okta:OktaDomain"),
-                ClientId = Configuration.GetValue<string>("Okta:ClientId"),
-                ClientSecret = Configuration.GetValue<string>("Okta:ClientSecret"),
-            });
+                {
+                    options.DefaultAuthenticateScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+                    options.DefaultSignInScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+                    options.DefaultChallengeScheme = CookieAuthenticationDefaults.AuthenticationScheme;
```

Comment:
Even I've changed the `DefaultChallengeScheme`, in the `SignIn` method I'm still using the Okta scheme:
```
Challenge(properties, OktaDefaults.MvcAuthenticationScheme)
```